mlos_bench/mlos_bench/config/optimizers/one_shot_opt.jsonc

```diff
@@ -20,12 +22,12 @@
             // In which case, unspecified values are governed by the "use_default" config parameter.
             "tunables/tunable-values-example.jsonc"
         ],
-        // Additional values of the tunable parameters.
-        "tunables_values": {},
+        // Additional values of the tunable parameters, encoded as a list of dictionaries.
+        "tunable_values": [{}],
```

Comment:
- [ ] Need to fix up the code to handle list of tunable params to test instead of just one.

Reply:
Will do that in a future PR - #344